### PR TITLE
WIP Break up the examples page into 4 pages

### LINKED ISSUE #904: Break up the examples page into 4 pages
We should break up the examples page into 4 separate pages (one for each section on the page).
https://www.kubeflow.org/docs/examples/resources/

Also add an _index page, following the pattern of the other doc directories.

This will solve 2 problems:

* The examples page is getting a little long.
* The link to the examples page in the left-hand nav looks odd, because all other links are to sections and are therefore bold.

### PR DESCRIPTION
**Pull request by mameshini**
fixes #904

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/918)

### CODE REVIEW

File: content/docs/examples/kubeflow-samples.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  url="https://github.com/kubeflow/examples/tree/master/pytorch_mnist" %}}
+Train a distributed PyTorch model on GCP and serve the model with Seldon Core.
+{{% /blocks/content-item %}}
+
+{{% blocks/content-item title="Ames housing value prediction"
+  url="https://github.com/kubeflow/examples/tree/master/xgboost_ames_housing" %}}
+Train an XGBoost model using the Kaggle Ames Housing Prices prediction on GCP.
+Use Seldon Core to serve the model locally, or GCP to serve it in the cloud.
+{{% /blocks/content-item %}}
+{{< /blocks/content-section >}}
```
Comment (kbhawkey) [inline content/docs/examples/kubeflow-samples.md]: Is this missing the matching opening shortcode?

File: content/docs/examples/blog-posts.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  url="https://towardsdatascience.com/how-to-create-and-deploy-a-kubeflow-machine-learning-pipeline-part-1-efea7a4b650f" %}}
+ A series of articles that walk you through the process of taking an existing
+ real-world TensorFlow model and operationalizing the training, evaluation,
+ deployment, and retraining of that model using Kubeflow Pipelines.
+ [Part 1](https://towardsdatascience.com/how-to-create-and-deploy-a-kubeflow-machine-learning-pipeline-part-1-efea7a4b650f)
+ (creating and deploying a pipeline), and
+ [part 2](https://towardsdatascience.com/how-to-deploy-jupyter-notebooks-as-components-of-a-kubeflow-ml-pipeline-part-2-b1df77f4e5b3)
+ (using Jupyter notebooks).
+{{% /blocks/content-item %}}
+{{< /blocks/content-section >}}
```
Comment (kbhawkey) [inline content/docs/examples/blog-posts.md]: Is the `blocks/content-section` needed? Check in all files.

**Comment by sarahmaddox** [comment]
Hallo @mameshini Please would you pull down the files from the Kubeflow/website repo again and fix the merge problems. The preview won't build until those are fixed.

File: content/docs/examples/blog-posts.md (hunk 2 above)
Comment (sarahmaddox) [reply content/docs/examples/blog-posts.md]: That's a good point. You can remove the `blocks/content-section` markup from all the files, and replace them with headings where necessary.

**Comment by sarahmaddox** [comment]
/assign @joeliedtke 

I'm assigning @joeliedtke to do a tech writing review.

File: content/docs/examples/_index.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
++++
+title = "Examples and tutorials"
```
Comment (joeliedtke) [inline content/docs/examples/_index.md]: Please change the title to: Examples and Tutorials

The Kubeflow style guide recommends using title case for headings. In title case, almost every word is capitalized with the exception of little words like: and, in, or.

File: content/docs/examples/blog-posts.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
++++
+title = "Blog posts"
```
Comment (joeliedtke) [inline content/docs/examples/blog-posts.md]: Please change the title to: Blog Posts

File: content/docs/examples/kubeflow-samples.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
++++
+title = "Kubeflow samples"
```
Comment (joeliedtke) [inline content/docs/examples/kubeflow-samples.md]: Please change the title to: Kubeflow Samples

File: content/docs/examples/blog-posts.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+description = "The following blog posts provide detailed examples and use cases. Be aware that
```
Comment (joeliedtke) [inline content/docs/examples/blog-posts.md]: Suggestion, move the content of the description to line 8, so it is a paragraph above the Kubeflow blog. 

Change the description to be something like:

Detailed examples and use cases

File: content/docs/examples/codelabs-tutorials.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
++++
+title = "Codelabs, Workshops, and Tutorials"
+description = "Below is a list of recommended end-to-end tutorials, workshops, walkthroughs,
```
Comment (joeliedtke) [inline content/docs/examples/codelabs-tutorials.md]: Suggestion, change the description to something like the following:

Recommended end-to-end tutorials, workshops, walk-throughs, and codelabs

File: content/docs/examples/_index.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+description = "A summary of recommended walkthroughs, blog posts, tutorials, and codelabs"
```
Comment (joeliedtke) [inline content/docs/examples/_index.md]: Change: walkthroughs -> walk-throughs

File: content/docs/examples/kubeflow-samples.md (hunk 1 above)
Comment (mameshini) [reply content/docs/examples/kubeflow-samples.md]: Fixed extra section blocks and created Videos page

File: content/docs/examples/videos.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
++++
+title = "Videos"
+description = "Tutorials and overviews published in video format. Be aware that a video
```
Comment (joeliedtke) [inline content/docs/examples/videos.md]: Suggestion, please move the description content to line 8. Please replace the description with something like:

Tutorial and overview videos

File: content/docs/examples/kubeflow-samples.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+description = "How to access the Kubeflow web UIs"
```
Comment (joeliedtke) [inline content/docs/examples/kubeflow-samples.md]: Please change the description to something like: 

Examples that demonstrate machine learning use cases with Kubeflow

**Pull request closed without merging**